pkg/rancher-components/src/components/Form/LabeledInput/LabeledInput.vue

```diff
@@ -301,6 +301,11 @@ export default (
       :value="tooltip"
       :status="status"
     />
+    <LabeledTooltip
```

Comment:
I don't think we can avoid a bunch of logic here but we can move some around to make it a bit less verbose. We still end up with two LabeledTooltips in the markup which I'm okay with since they serve two separate purposes and theoretically we might want to render both at the same time at some point. I did however combine the two LabeledTooltips that were already there since they were basically doing the same thing.